.eslintrc.example

```diff
@@ -3,16 +3,21 @@
     "browser": true,
     "amd": true,
     "es6": true,
-    "mocha": true,
+    "mocha": true
   },
+  "extends": ["standard", "standard-react", "standard-jsx"],
   "parser": "babel-eslint",
   "rules": {
     "strict": 0,
     "comma-dangle": [2, "always-multiline"],
-    "quotes": "single",
+    "jsx-quotes": [2, "prefer-double"],
+    "no-multi-spaces": 0,
     "no-underscore-dangle": 0,
+    "quotes": [2, "single"],
+    "semi": [2, "always"],
+    "space-before-function-paren": 0
```

Comment:
Not a fan of spaces between `function` and the opening paren, so would prefer "never" here.